### PR TITLE
Allow build-unix.sh to work on Arch Linux

### PR DESCRIPTION
**Pull request by dmp1ce**
Changes shell script and python script to allow `build-unix.sh` to work on Arch Linux based distros.

### CODE REVIEW

File: pyinstaller/specterd.spec [hunk 1]
```diff
@@ -14,7 +14,10 @@ elif platform.system() == 'Linux':
     if platform.processor() == 'aarch64': #ARM 64 bit
         binaries = [("/lib/aarch64-linux-gnu/libusb-1.0.so.0", ".")]
     else:
-        binaries = [("/lib/x86_64-linux-gnu/libusb-1.0.so.0", ".")]
+        if distro.name() == "Arch Linux":
+            binaries = [("/usr/lib/libusb-1.0.so.0", ".")]
+        else:
+            binaries = [("/lib/x86_64-linux-gnu/libusb-1.0.so.0", ".")]
```
Comment (stepansnigirev) [inline pyinstaller/specterd.spec:20]: I suggest doing it slightly differently:
to set up a list of candidates where the library can live in different linux distros, and try to find one:
```suggestion
        candidates = [
                "/usr/lib/libusb-1.0.so.0",
                "/lib/x86_64-linux-gnu/libusb-1.0.so.0",
                "/lib/aarch64-linux-gnu/libusb-1.0.so.0",
                "/lib/arm-linux-gnueabihf/libusb-1.0.so.0",
        ]
        binaries = []
        for p in candidates:
            if os.path.isfile(p):
                binaries = [(p, ".")]
                break
```

**Comment by stepansnigirev** [comment]
Looks good, just suggested one small tweak to make it more distro-agnostic.
See above.

**Comment by dmp1ce** [comment]
Updated PR with suggestion.

**Comment by dmp1ce** [comment]
Removed distro import.

**Comment by stepansnigirev** [comment]
Thanks! Merging.

**Pull request merged**